neo/io/nwbio.py

```diff
@@ -458,15 +476,16 @@ def write_all_blocks(self, blocks, **kwargs):
                 'block', 'the name of the Neo Block to which the SpikeTrain belongs')
 
         if sum(statistics(block)["Epoch"]["count"] for block in blocks) > 0:
-            nwbfile.add_epoch_column('_name', 'the name attribute of the Epoch')
-            # nwbfile.add_epoch_column('_description', 'the description attribute of the Epoch')
-            nwbfile.add_epoch_column(
-                'segment', 'the name of the Neo Segment to which the Epoch belongs')
-            nwbfile.add_epoch_column('block',
-                                     'the name of the Neo Block to which the Epoch belongs')
+            nwbfile.add_trial_column('segment', 'name of the Segment to which the Epoch belongs')
+            nwbfile.add_trial_column('block', 'name of the Block to which the Epoch belongs')
```

Comment:
if nwb is supporting the `epoch` concept, what is the reason to switch to `trials` now?

Reply:
Yes. You are right. As it supports the epoch concept, it is not necessary to have the trials function.
I put it back as before.